exercises/rail-fence-cipher/rail_fence_cipher_test.rb

```diff
@@ -0,0 +1,94 @@
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'rail_fence_cipher'
+
+# rubocop:enable all
+class RailFenceCipherTest < Minitest::Test
+  def test_encode_with_empty_string
+    assert_equal '', RailFenceCipher.encode('', 4)
+  end
+
+  def test_encode_with_one_rail
+    skip
+    assert_equal 'One rail, only one rail',
+                 RailFenceCipher.encode('One rail, only one rail', 1)
+  end
+
+  def test_encode_with_two_rails
+    skip
+    assert_equal 'XXXXXXXXXOOOOOOOOO',
+                 RailFenceCipher.encode('XOXOXOXOXOXOXOXOXO', 2)
+  end
+
+  def test_encode_with_three_rails
+    skip
+    assert_equal 'WECRLTEERDSOEEFEAOCAIVDEN',
+                 RailFenceCipher.encode('WEAREDISCOVEREDFLEEATONCE', 3)
+  end
+
+  def test_encode_with_ending_in_the_middle
+    skip
+    assert_equal 'ESXIEECSR', RailFenceCipher.encode('EXERCISES', 4)
+  end
+
+  def test_encode_with_a_more_diverse_text
```

Comment:
I think so, when I see a test I always wonder what feature/edge case it is covering. If it is not doing this I usually remove it since it is more code for people to read that is not helping them to understand the logic.